### ISSUE TITLE
well , it looks that you do not have in your sources.list any repository from kali .

### ISSUE THREAD

**Issue by the-mcnaveen** [Screetsec/TheFatRat#349]
well , it looks that you do not have in your sources.list any repository from kali .
open this file with any text editor : sources.list in the folder /etc/apt

add these lines to the beginning : 

deb http://http.kali.org/kali kali-rolling main contrib non-free
deb-src http://http.kali.org/kali kali-rolling main contrib non-free
deb http://old.kali.org/kali sana main non-free contrib
deb-src http://old.kali.org/kali sana main non-free contrib

save the file and then write this in your terminal : 
apt-get update && apt-get upgrade -y

then run setup of fatrat again .

Basically you would never be able to update fatrat or install anything new .
http://docs.kali.org/general-use/kali-linux-sources-list-repositories

This issue you have is not related to fatrat .

_Originally posted by @peterpt in https://github.com/Screetsec/TheFatRat/issues/126#issuecomment-305626835_

**Comment by NaheelSalman** [comment]
@the-mcnaveen where is the issue???
i mean i forgot what i had posted 
you gave solution that's pretty appreciated but now help me in finding issue or mention issue number

**Comment by the-mcnaveen** [comment]
Hey, Here is the issue #350
